src/main/java/com/github/tombentley/klog/segment/reader/SegmentDumpReader.java

```diff
@@ -373,13 +375,14 @@ private File readDumpingLine(String line) {
         if (!dumpingMatcher.matches()) {
             throw new UnexpectedFileContent("Expected first line to match " + dumpingPattern);
         } else {
-            segmentFile = new File(dumpingMatcher.group(1));
+            Path path = Paths.get(dumpingMatcher.group(1));
+            segmentFile = path.toFile();
         }
         return segmentFile;
     }
 
     private long readStartingOffsetLine(File segmentFile, String line) {
-        String startingOffsetPattern = "^Starting offset: ([0-9]+)$";
+        String startingOffsetPattern = "^(?i).*starting offset: ([0-9]+)$";
```

Comment:
It would be less cryptic to use the `Pattern.CASE_INSENSITIVE` flag in the call to `compile` on the following line.